### ISSUE TITLE
Why using TouchableWithoutFeedback instead Pressable?

### ISSUE THREAD

**Issue by davidnum** [nandorojo/moti#133]
Hello.
Why using Why using TouchableWithoutFeedback instead Pressable?
https://github.com/nandorojo/moti/blob/master/packages/interactions/src/pressable/pressable.tsx

**Comment by nandorojo** [comment]
I believe the gesture handler has better native performance, so I went with that

**Comment by davidnum** [comment]
It's create much problems with "pressability" and styling. Example:  TouchableWithoutFeedback not working in modals on Android.

**Comment by nandorojo** [comment]
Hm why doesn’t it work there? Maybe you need to wrap your app with the gesture handler root view? Or wrap the modal in the gesture handler root view?

**Comment by davidnum** [comment]
> 

I mean that i should wrap modal with HOC to use MotiPressable in modals on android. It would be nice to be able to replace main pressable component when it necessary. Something like this:
```jsx
import { Pressable } from 'react-native' 

<MotiPressable PressableComponent={Pressable} />

```

**Comment by nandorojo** [comment]
Maybe try gesture handler v2? There is a breaking change in v2 where you have to wrap your app in the RNGH root view, so be aware of that. If you’re using expo, this is available in the expo 44 beta. I wouldn’t upgrade to reanimated 2.3 quite yet though.

**Comment by nandorojo** [comment]
Yeah, I’ve considered that API. But I’m a bit hesitant, since it shouldn’t technically be necessary. The bug you’re mentioning is a RNGH bug that is supposed to be fixed in v2. Maybe there’s a valid use-case for a custom `Pressable` component, but I’m not certain I’ve seen one.

**Comment by nandorojo** [comment]
Checking if you tried with v2

**Comment by nandorojo** [comment]
will re-open if there's more info
